### ISSUE TITLE
Change training script to not apply weight decay to LayerNorm or biases

### ISSUE THREAD

**Issue by rtaylor-rx-m** [dingo-actual/om#12]
Currently weight decay is applied to all parameters.

**Comment by dingo-actual** [comment]
Fixed
